malaba6/E-Rental
# #167148017 built sign-up page

#### What does this PR do?
It creates a sign-up page.

#### Description of Task to be completed?
1. It creates and adds a sign-up page to the sign-up button on the homepage.
2. It adjusts some HTML and CSS of the homepage to maintain consistency.

#### How should this be manually tested?
Navigate to the homepage page and click on the **Sign Up** button and check if you get the to the registration form. 

#### Secreenshots

**Desktop Version**

![sign-up html](https://user-images.githubusercontent.com/34879554/61395464-c9d95d00-a8c5-11e9-83c2-d7b277738786.png)


**Mobile Version**

![sign-up-mobile html](https://user-images.githubusercontent.com/34879554/61395514-e4abd180-a8c5-11e9-8c95-15c546730871.png)
